### PR TITLE
Getting dotdotdot to work

### PR DESCRIPTION
**Pull request by mbad0la**

### CODE REVIEW

**Comment by mbad0la** [comment]
@Slayslot this PR is in reference to #54  
Let's resolve it asap! :)

**Comment by mbad0la** [comment]
@Slayslot , please have a look. I guess things look decent.

**Comment by handasolo** [comment]
@mbad0la Seems to be very buggy as of now. Let me explicitly tell the ones I've found so far.
1. Open the website and go to more past events. It will lead to the following screenshot. I'm just putting in one but it happens for all of the cards.
screen shot 2016-04-09 at 16 54 35
2. Follow step one and then press the back button. You'll see the following as soon as you enter responsive mode(I've no idea why responsive mode triggers this.)
screen shot 2016-04-09 at 16 57 35
3. Directly load `$Working_Dir/index.html#past-more` onto your browser(Where `$Working_Dir` is the directory where your repo is located. You'll see the same problem as in point number 1. Now, hit the back button and you'll see the following.
screen shot 2016-04-09 at 17 01 18

I guess we should take the route @h4xr was taking since this is turning out to be so buggy. What do you say @mbad0la ?

**Comment by mbad0la** [comment]
I am unaware of what @h4xr suggested. Please let me know about that?  
At the same time, let me try to resolve these bugs.

**Comment by handasolo** [comment]
@mbad0la Take a look at how things are done in #75

**Comment by mbad0la** [comment]
@Slayslot the route @h4xr was taking was implementing dotdotdot behaviour on the title. And he would have come across the same problems later in the implementation that i've come across now.  
Anyway, i pushed in some minor changes. Can you please let me know about any positive outcome?

**Comment by handasolo** [comment]
@mbad0la 1 and 2 seems to be fixed, 3 still prevails though. Meanwhile, also add it to the location as it is added for the title.

**Comment by mbad0la** [comment]
@Slayslot have a look at the changes.  
pacerbug2

**Comment by mbad0la** [comment]
@Slayslot did you check out the changes?

**Comment by handasolo** [comment]
Saw your GIF, still have to look at the code though.

**Comment by handasolo** [comment]
@mbad0la Sorry for the delay, I've been away from linux these past days and might be away from it for a few more sadly. Anyhow, This seems to work really well now. One issue I forgot to bring up is that you changed the size of font-size of location on small screen. Is that necessary? Kind of undermines the importance of location on small screens.

Anyhow, this looks good to me. However, seeing this was causing so many problems I wouldn't want this to come back and bite us somehow. So, I'm going to ask another person from the community to come and take a look. Thanks for all your work @mbad0la.

**Comment by mbad0la** [comment]
@Slayslot I kind of reduced the font-size to improve readability, but i guess we can change it back to the original size if you think that would be best. And if i can recall correctly, the original font-size was taking too much space on the headings and there wasn't much space for the main content.  
Anyway, it would be great to get this cross-checked.

File: src/client/css/style.css [hunk 1]
```diff
@@ -427,9 +427,9 @@ section#events {
   color: #fff;
 }
 
-.ellipsis {
+/*.ellipsis {
   overflow: hidden;
-}
+}*/
```
Comment (manrajgrover) [inline src/client/css/style.css]: Please remove this piece of code.

File: src/client/css/style.css [hunk 2]
```diff
@@ -475,6 +475,12 @@ section#events {
   bottom: unset;
 }
 
+.cs-style-3 figure:hover figcaption span
+{
```
Comment (manrajgrover) [inline src/client/css/style.css]: Nit. Please keep brackets on same line with single space between selector and `{`.

File: src/client/css/style.css [hunk 3]
```diff
@@ -615,9 +627,9 @@ section#events {
 }
 
 @media (max-width:363px) {
-  .cs-style-3 figure:hover figcaption p {
+  /*.cs-style-3 figure:hover figcaption p {
       height: 100px;
-  }
+  }*/
```
Comment (manrajgrover) [inline src/client/css/style.css]: Please remove this piece of code as well.

File: src/client/js/script.js [hunk 4]
```diff
@@ -65,8 +65,8 @@ function checkDynamicVScroll(){
   }else if(testcaseheight > windowheight && !FIREFOX){
     $('#outer-wrapper').removeClass('no-margin-wrapper');
   }
-  console.log(testcaseheight);
-  console.log(windowheight);
+  //console.log(testcaseheight);
+  //console.log(windowheight);
```
Comment (manrajgrover) [inline src/client/js/script.js]: Please remove comments.

File: src/client/js/script.js [hunk 5]
```diff
@@ -480,9 +488,9 @@ $(document).ready(function() {
   });
   //Nav End
   //Events Section start
-  $(".ellipsis").dotdotdot({
+  /*$(".ellipsis").dotdotdot({
     watch: "window"
-  });
+  });*/
```
Comment (manrajgrover) [inline src/client/js/script.js]: Please remove comments.

**Comment by manrajgrover** [comment]
@mbad0la Hey! First of all thank you for working on this. 

Things look good to go. Please fix final nits I have left. @Slayslot can review it and land this fix.

**Comment by handasolo** [comment]
Also, restore the font size. We can have a separate issue related to that. @mbad0la

**Comment by mbad0la** [comment]
@Slayslot @ManrajGrover confirm cleanup. Let me know if I missed something :)

**Comment by handasolo** [comment]
Looks good to me. Merging. Thanks for all the effort @mbad0la .

**Comment by manrajgrover** [comment]
Looks good to me as well. Thanks @mbad0la :blush:

**Comment by CuriousLearner** [comment]
Thanks @mbad0la 

cc @Slayslot

**Pull request merged**